### PR TITLE
Improvments!

### PR DESCRIPTION
**Pull request by LEGOlord208**

### CODE REVIEW

File: Source/MCToolkit.cpp [hunk 1]
```diff
@@ -1,31 +1,79 @@
...
-#include <Windows.h>
 #include <string>
+#include <signal.h>
 using namespace std;
 
-void main()
-{
-	SetConsoleTitle(TEXT("MC Server Toolkit"));
+#ifdef OS_WINDOWS
+	#include <Windows.h>
```
Comment (tbodt) [inline Source/MCToolkit.cpp]: The ***almighty convention*** is to not indent things inside preprocessor directives.

Comment (LEGOlord208) [inline Source/MCToolkit.cpp]: Then you either wait until tomorrow, or you merge it and fix it later

**Comment by tbodt** [comment]
¯\_(ツ)_/¯

**Comment by kittyhacker101** [comment]
I don't think this is needed. This is a program only made for Windows. Also, why did you remove the console title?

**Comment by tbodt** [comment]
It's not so easy to set the console title on Unix.

**Comment by kittyhacker101** [comment]
So? This program is only for Windows.

**Comment by LEGOlord208** [comment]
I didn't remove the console title!?

**Comment by LEGOlord208** [comment]
Wait, yeah I did! Sorry about that, I'll fix that too. It was an accident

**Comment by LEGOlord208** [comment]
There we go

**Comment by LEGOlord208** [comment]
Any update on this? @kittyhacker101

**Comment by kittyhacker101** [comment]
Well, I am quite a bit busy working on a new project (KatWeb, a web server written by me), so it might be quite a while before I can compile and release this.

**Comment by LEGOlord208** [comment]
You can still merge it...

**Comment by kittyhacker101** [comment]
Fine, I will merge it.

**Comment by LEGOlord208** [comment]
You closed it... That's not how it works...

**Comment by tbodt** [comment]
fixed

**Comment by kittyhacker101** [comment]
Oops, I clicked the wrong button :P

**Pull request merged**